style.css

```diff
@@ -1,3 +1,16 @@
+
+h1 {
+    font-family: ‘Metrophobic’, Arial, serif; font-weight: 400;
+}
+
+p {
+    font-family: ‘Metrophobic’, Arial, serif; font-weight: 400;
```

Comment:
No need to duplicate this, just make the h1 rule apply to p elements as well `h1, p { ... }`